spec/unit/form_builder_spec.rb

```diff
@@ -277,6 +277,43 @@ def build_form(options = {}, form_object = Post.new, &block)
 
   end
 
+  context "with a has_one relation on an author's profile" do
+    let :body do
+      author = user()
+      build_form do |f|
+        f.inputs do
+          f.input :title
+          f.input :body
+        end
+        f.form_builder.instance_eval do
+          @object.author = author
+        end
+        f.inputs name: 'Author', for: :author do |author|
```

Comment:
Shadowing outer local variable - `author`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.